src/main/java/Drew.java

```diff
@@ -0,0 +1,309 @@
+import drew.exceptions.InsufficientArgumentsException;
+import drew.exceptions.UnknownCommandException;
+import drew.tasktypes.Deadline;
+import drew.tasktypes.Event;
+import drew.tasktypes.Task;
+import drew.tasktypes.Todo;
+
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Main chatbot class. Contains the logic of the chatbot.
+ *
+ * @author cocoanautz
+ */
+public class Drew {
+    /**
+     * Checks the identity of the command.
+     *
+     * @param input String containing full user input.
+     * @return Command specified by user input.
+     */
+    public static Command checkCommandIdentity(String input) throws UnknownCommandException,
```

Comment:
Avoid complicated expressions and definitions. Your method `checkCommandIdentity` gets the userCommand by checking for string equivalence, then uses the userCommand to perform a conditional switch case check. This can be combined into 1 and reduce redundant checkings and simplify your method to make it more easily readble